### ISSUE TITLE
Installation Error - DandbCompany.SalesVolume does not exist

### ISSUE THREAD

**Issue by yochevreux** [alexed1/ProcessBuilderConverter#16]
Hi @alexed1, I encoutered an error while installing the package.

Demo_Process_A_Recursion: formula_myRule_1 (Formula) - The formula expression is invalid: Field myVariable_current.DandbCompany.SalesVolume does not exist. Check spelling. 

It seems that Data.com objects and fields are removed from orgs when Data.com licences are expired.

_Originally posted by @yochevreux in https://github.com/alexed1/ProcessBuilderConverter/issues/1#issuecomment-918016453_

**Comment by alexed1** [comment]
Can you think of anything that can be done in this situation? I can't think of anything...

**Comment by yochevreux** [comment]
Nope, I was secretly hoping it would have rang a bell for you. I'll check on other orgs seeded from the same production org to see if there is the same issue.
BTW, thank you very much for your work and for sharing it.
Yohann

**Comment by yochevreux** [comment]
@alexed1 I tried to install version 1.1 instead of 1.2 even if the org is in Winter 22, and it worked.

**Comment by alexed1** [comment]
try 1.2.1 it shouldn't have the DandB problem.
